modules/private-cluster/outputs.tf

```diff
@@ -78,6 +78,11 @@ output "ca_certificate" {
   value       = "${local.cluster_ca_certificate}"
 }
 
+output "enable_binary_authorization" {
```

Comment:
Please remove this output; it doesn't provide any value.

Reply:
Ok, done